docs/_articles/en/einstein/einstein-overview.md

```diff
@@ -13,11 +13,13 @@ Einstein for Developers assists you throughout the Salesforce development proces
 
 ## Current Capabilities
 
-This release of Einstein for Developers focuses on boilerplate Apex code generation from natural language prompts. This feature, used along with IntelliSense, makes Apex development tooling in Visual Studio Code even richer. Familiarity with Visual Studio Code is assumed.
+Use Einstein for Developers to generate boilerplate Apex code from natural language prompts and to automatically receive code completion suggestions inline. Used along with IntelliSense, this feature makes Apex development tooling in Visual Studio Code even richer. Familiarity with Visual Studio Code is assumed.
```

Comment:
```suggestion
Einstein for Developers generates Apex code from natural language prompts and automatically suggests code completions for you as you type. When enabled along with IntelliSense, this feature makes Apex development tooling in Visual Studio Code even richer. Familiarity with Visual Studio Code is assumed.
```
Can we make it seem more like E4D is _doing something for you_ rather than you are _using_ E4D?  I like thinking of Inline Completions like Einstein is a person/thing that knows what I need and is trying to help me out and anticipate what I want.